vcd/resource_vcd_vapp_multi_vm_in_template_test.go

```diff
@@ -0,0 +1,156 @@
+// +build vapp vm ALL functional
+
+package vcd
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+func TestAccVcdVAppMultiVmInTemplate(t *testing.T) {
+	var vapp govcd.VApp
+	var vm govcd.VM
+	vappName := t.Name()
+	vmName := t.Name() + "VM"
+	vmName2 := t.Name() + "VM2"
+	var params = StringMap{
+		"Org":                testConfig.VCD.Org,
+		"Vdc":                testConfig.VCD.Vdc,
+		"EdgeGateway":        testConfig.Networking.EdgeGateway,
+		"NetworkName":        "TestAccVcdVAppVmNet",
+		"Catalog":            testConfig.VCD.Catalog.Name,
+		"CatalogItemMultiVm": testConfig.VCD.Catalog.CatalogItemWithMultiVms,
+		"VmNameInTemplate":   testConfig.VCD.Catalog.VmName1,
+		"VmNameInTemplate2":  testConfig.VCD.Catalog.VmName2,
+		"VappName":           vappName,
+		"VmName":             vmName,
+		"VmName2":            vmName2,
+		"ComputerName":       vmName + "-unique",
+		"Tags":               "vapp vm",
+	}
+
+	configText := templateFill(testAccCheckVcdVAppVmMultiVmInTemplate, params)
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	if testConfig.VCD.Catalog.CatalogItemWithMultiVms == "" || testConfig.VCD.Catalog.VmName1 == "" || testConfig.VCD.Catalog.VmName2 == "" {
```

Comment:
I think this check must be above `templateFill`. Otherwise it will generate binary test with empty variables when one doesn't have image specified

Reply:
ty. moved